test/features/pagecontent/pagecontent.js

```diff
@@ -21,8 +21,8 @@ describe('item requests', function() {
         .then(function(res) {
             assert.deepEqual(res.status, 200);
             assert.deepEqual(res.headers['access-control-allow-origin'], '*');
-            assert.deepEqual(res.headers['access-control-allow-methods'], 'GET');
-            assert.deepEqual(res.headers['access-control-allow-headers'], 'accept, content-type, cache-control, ' +
+            assert.deepEqual(res.headers['access-control-allow-methods'], 'GET,HEAD');
+            assert.deepEqual(res.headers['access-control-allow-headers'], 'accept, content-type, content-length, cache-control, ' +
```

Comment:
Let's test for `referrer-policy` as well since we're testing for all the CORS headers here.